entries/c/commercial.metrobankonline.co.uk.json

```diff
@@ -0,0 +1,25 @@
+{
+  "Metro Bank Commercial": {
+    "domain": "commercial.metrobankonline.co.uk",
+    "url": "https://metrobankonline.co.uk",
+    "img": "metrobankonline.co.uk.svg",
+    "tfa": [
+      "custom-hardware",
+      "custom-software"
+    ],
+    "custom-software": [
+      "Metro Bank Authenticator app"
+    ],
+    "custom-hardware": [
+      "Security Device"
+    ],
+    "documentation": "https://www.metrobankonline.co.uk/help-and-support/what-is-sca/",
```

Comment:
Sounds like they only prompt for OTP codes on transactions and not login attempts.

Reply:
From what I can tell, the `custom-software` methods can be used for authentication, rather than authorisation:

https://www.metrobankonline.co.uk/help-and-support/what-is-sca/#item17066
> It’s an app you can use instead of your security device to prove it’s you when logging in or making changes on your Business Online Plus and Commercial Online Banking. You can use the app to:
> 
> log in using a different device or browser
> make certain payments and transactions including online shopping
> make admin preference/access changes
> The app is only available to those with a UK mobile number.